.github/workflows/setup-dev-drive.ps1

```diff
@@ -0,0 +1,24 @@
+# This creates a 20GB dev drive, and exports all required environment
+# variables so that rustup, uv and others all use the dev drive as much
+# as possible.
+$Volume = New-VHD -Path C:/uv_dev_drive.vhdx -SizeBytes 20GB |
+					Mount-VHD -Passthru |
+					Initialize-Disk -Passthru |
+					New-Partition -AssignDriveLetter -UseMaximumSize |
+					Format-Volume -FileSystem ReFS -Confirm:$false -Force
+
+Write-Output $Volume
+
+$Drive = "$($Volume.DriveLetter):"
+$Tmp = "$($Drive)/uv-tmp"
+
+Write-Output `
+	"DEV_DRIVE=$($Drive)" `
+	"TMP=$($Tmp)" `
+	"TEMP=$($Tmp)" `
+	"UV_INTERNAL__TEST_DIR=$($Tmp)" `
```

Comment:
Was `UV_INTERNAL__TEST_DIR` going to be removed as part of this PR?

Reply:
I think this is addressed in https://github.com/astral-sh/uv/pull/6858#issuecomment-2320839792

Reply:
I see, thanks

Reply:
it can't be:

https://github.com/astral-sh/uv/pull/6858#issuecomment-2320839792